### PR TITLE
[DOCFIX] Improve troubleshooting doc

### PR DESCRIPTION
**Pull request by jiacheliu3**
Minor improvements like typo fixed

### CODE REVIEW

File: docs/en/operation/Troubleshooting.md [hunk 1]
```diff
@@ -308,7 +310,7 @@ A: Alluxio accelerates your system performance by leveraging temporal or spatial
 Alluxio can be configured under a variety of modes, in different production environments.
 Please make sure the Alluxio version being deployed is update-to-date and supported.
 
-When posting questions on the [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users),
+When posting questions on the [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users) or Slack channel,
```
Comment (gpang) [inline docs/en/operation/Troubleshooting.md:313]:
```suggestion
When posting questions on the [Mailing List](https://groups.google.com/forum/#!forum/alluxio-users) or [Slack channel](https://slackin.alluxio.io/),
```

Comment (jiacheliu3) [reply docs/en/operation/Troubleshooting.md]: Sorry I missed this comment

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**